$OpenBSD$

Avoid _XOPEN_SOURCE as it breaks build of time_zone_format.cc.

Index: absl/time/internal/cctz/src/time_zone_format.cc
--- absl/time/internal/cctz/src/time_zone_format.cc.orig
+++ absl/time/internal/cctz/src/time_zone_format.cc
@@ -18,12 +18,6 @@
 #endif
 #endif
 
-#if defined(HAS_STRPTIME) && HAS_STRPTIME
-#if !defined(_XOPEN_SOURCE)
-#define _XOPEN_SOURCE  // Definedness suffices for strptime.
-#endif
-#endif
-
 #include "absl/base/config.h"
 #include "absl/time/internal/cctz/include/cctz/time_zone.h"
 
